subgraphs/uniswap-v3/README.md

```diff
@@ -38,19 +47,19 @@ Note that Pool Fee Tiers vary by pool and more tiers could be added by Governanc
 
 ### Total Unique Users
 
-Count of  Unique Addresses which have interacted with the protocol via any transaction
+Count of Unique Addresses which have interacted with the protocol via any transaction
 
 `Swaps`
 
 `Deposits`
 
 `Withdraws`
 
-###  Reward Token Emissions Amount
+### Reward Token Emissions Amount
 
 To be added
```

Comment:
should probably update this and Protocol Controlled Value (does not apply).

Reply:
Just add does not apply like so? Or do you think it's better to just remove them?